core/Network/TLS/Extra/Cipher.hs

```diff
@@ -652,6 +655,28 @@ cipher_ECDHE_RSA_AES256GCM_SHA384 = Cipher
     , cipherMinVer       = Just TLS12 -- RFC 5289
     }
 
+cipher_AES128GCM_SHA256 :: Cipher
+cipher_AES128GCM_SHA256 = Cipher
+    { cipherID           = 0x1301
+    , cipherName         = "AES128GCM-SHA256"
+    , cipherBulk         = bulk_aes128gcm
+    , cipherHash         = SHA256
+    , cipherPRFHash      = Nothing
+    , cipherKeyExchange  = CipherKeyExchange_TLS13
+    , cipherMinVer       = Just TLS13ID16
```

Comment:
Following up, I see no `TLS13ID16` anywhere in the existing code base.  So this commit seems to depend on other commits that are not in this pull request.  I think the right thing here is to create a tls13-dev branch, and use `TLS13` as the (final) version number, even though the TLS 1.3 RFC is not yet published.